tests/e2e/cucumber/features/app-provider/lock.feature

```diff
@@ -16,8 +16,8 @@ Feature: lock
       | resource | type         | content      |
       | test.odt | OpenDocument | some content |
     And "Alice" shares the following resource using API
-      | resource | recipient | type | role     |
-      | test.odt | Brian     | user | Can edit |
+      | resource | resourceType | recipient | type | role     | space    |
+      | test.odt | file         | Brian     | user | Can edit | Personal |
```

Comment:
I wonder if we can do the same with `resourceType` here as we did for `space`

Reply:
okay i will try